helm/values/explorer-goquorum.yaml

```diff
@@ -0,0 +1,64 @@
+image:
+  explorer:
+    repository: consensys/quorum-explorer
+    tag: latest
+    pullPolicy: Always #IfNotPresent
+
+resources:
+  requests:
+    cpu: 1000m
+    memory: 1G
+
+explorerConfig: |
```

Comment:
Prob add some comments in here saying the accounts are our test accounts, nothing to worry about :)